trap/src/main/java/com/cursorinsight/trap/TrapManager.kt

```diff
@@ -270,14 +249,38 @@ class TrapManager internal constructor(
         }
     }
 
+    private fun getLowBatteryStatus() : Boolean {
```

Comment:
Does it make sense to share code between this function and TrapBatteryCollector's `processBatteryStatus()`?

Reply:
As we are subscribing on different events in the Manager and Collector and the collector reads more properties from the intent I don't see any easy way to do it.